standard.md

```diff
@@ -6,17 +6,17 @@
+**4. Platform Independence** | When the digital public good has mandatory dependencies that create more restrictions than the original license, proving independence from the closed component(s) and/or indicating the existence of functional, open alternatives that can be used without significant changes to the core product is required.
+**5. Documentation** | Digital public goods require documentation of the source code, use cases, and/or functional requirements. For content collections, this should include all relevant/compatible apps, software, or hardware required to access the content collection, and instructions regarding how to use it. For software solutions, this should be technical documentation that would allow a technical person unfamiliar with the project to launch and run the software. For data projects, this should be documentation that describes all the fields in the set, and provides context on how the dataset was collected, and how it should be interpreted.
+**6. Mechanism for Extracting Data and Content** | Digital Public Goods with non personally identifiable information (PII) design for possibility of extracting or importing non-PII data and content from the system in a non-proprietary format.
+**7. Adherence to Privacy and Applicable Laws** | Digital Public Goods must be designed and developed to comply with applicable privacy laws.
+**8. Adherence to Standards & Best Practices** | Digital Public Goods must be designed and developed to align with relevant standards, best practices, and/or principles. For example,[the Principles for Digital Development](https://digitalprinciples.org/principles/).
+**9. Do No Harm By Design** | Digital Public Goods must be designed to anticipate, prevent, and do no harm by design.
+**9a) Data Privacy & Security** | Digital Public Goods that collect data must indicate the types of data collected and stored. DPGs must demonstrate how they are designed to ensure that the privacy and security of this data, in addition to how to the design of the DPG mitigates and prevents adverse impacts that can result from data collection, storage and distribution.
+**9b) Inappropriate & Illegal Content** | Digital Public Goods designed to collect, store or distribute content collections must have policies for identifying inappropriate, harmful and illegal content collections (i.e child sexual abuse materials) in addition to mechanisms for detecting, moderating and removing inappropriate/illegal content.
+**9c) Protection from Harassment** | When Digital Public Goods facilitate interactions with or between users or contributors, they are required to have a mechanism for users and contributors to protect themselves against grief, abuse, and harassment. In addition, DPGs must have a mechanism to address the safety and security of underage users. 
```

Comment:
Originally, the content in 9(c) was designed to refer to interactions that happen outside of the scope of the DPG product(s) in places like online forums, chat rooms, mailing lists, etc. As such, processes and policies would not generally exist within the DPG as part of the DPG software itself, but rather through the community project's other non-product artifacts such as community participation guidelines, codes of conduct, terms of service/use, privacy policies, trademark policies, etc.

Reply:
Good point. However, this is why we are not prescriptive of the mechanism, but simply mention that there must be one. The standards do ask for both code and governance measures since we understand, as you pointed out, that code alone is not sufficient.